### PR TITLE
Update the French translation

### PR DESCRIPTION
**Pull request by zenseii**
Unfuzzy hero strings, add new translations

### CODE REVIEW

File: files/lang/fr.po [hunk 1]
```diff
@@ -2682,21 +2679,20 @@ msgstr "sans artéfact."
...
 
-#, fuzzy
 msgid ""
 "'Grouped' combat formation bunches the castle's units together in the center "
 "of the castle's side of the battlefield."
 msgstr ""
-"La formation de combat 'groupée' regroupe votre armée au centre du champ de "
-"bataille, de votre coté."
+"La formation de combat 'groupée' regroupe les troupes du château au centre "
+"du champ de bataille, du coté du château."
```
Comment (FlorianBaba) [inline files/lang/fr.po:2695]:
```suggestion
"du champ de bataille, à l'intérieur du château."
```

File: files/lang/fr.po [hunk 2]
```diff
@@ -4509,19 +4502,19 @@ msgid "hotkey|select map"
 msgstr "Sélectionner carte"
 
 msgid "hotkey|select small map size"
-msgstr ""
+msgstr "Voir les mappes de taille petite"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4505]:
```suggestion
msgstr "Voir les cartes de taille petite"
```

File: files/lang/fr.po [hunk 3]
```diff
@@ -4509,19 +4502,19 @@ msgid "hotkey|select map"
...
 
 msgid "hotkey|select medium map size"
-msgstr ""
+msgstr "Voir les mappes de taille moyenne"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4508]:
```suggestion
msgstr "Voir les cartes de taille moyenne"
```

File: files/lang/fr.po [hunk 4]
```diff
@@ -4509,19 +4502,19 @@ msgid "hotkey|select map"
...
 
 msgid "hotkey|select large map size"
-msgstr ""
+msgstr "Voir les mappes de taille grande"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4511]:
```suggestion
msgstr "Voir les cartes de taille grande"
```

File: files/lang/fr.po [hunk 5]
```diff
@@ -4509,19 +4502,19 @@ msgid "hotkey|select map"
...
 
 msgid "hotkey|select extra large map size"
-msgstr ""
+msgstr "Voir les mappes de taille très grande"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4514]:
```suggestion
msgstr "Voir les cartes de taille très grande"
```

File: files/lang/fr.po [hunk 6]
```diff
@@ -4509,19 +4502,19 @@ msgid "hotkey|select map"
...
 
 msgid "hotkey|select all map sizes"
-msgstr ""
+msgstr "Voir toutes les mappes"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4517]:
```suggestion
msgstr "Voir toutes les cartes"
```

File: files/lang/fr.po [hunk 7]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
 msgstr "Recommencer le scénario de campagne"
 
 msgid "hotkey|world map left"
-msgstr ""
+msgstr "Déplacement vers la gauche dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4589]:
```suggestion
msgstr "Déplacement vers la gauche sur la carte du monde"
```

File: files/lang/fr.po [hunk 8]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map right"
-msgstr ""
+msgstr "Déplacement vers la droite dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4592]:
```suggestion
msgstr "Déplacement vers la droite sur la carte du monde"
```

File: files/lang/fr.po [hunk 9]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map up"
-msgstr ""
+msgstr "Déplacement vers le haut dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4595]:
```suggestion
msgstr "Déplacement vers le haut sur la carte du monde"
```

File: files/lang/fr.po [hunk 10]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map down"
-msgstr ""
+msgstr "Déplacement vers le bas dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4598]:
```suggestion
msgstr "Déplacement vers le bas dans la carte du monde"
```

File: files/lang/fr.po (hunk 8 above)
Comment (FlorianBaba) [reply files/lang/fr.po:4592]: "carte" could also be used IMO (instead of "carte du monde")

File: files/lang/fr.po [hunk 11]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map up left"
-msgstr ""
+msgstr "Déplacement vers le haut et la gauche dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4601]:
```suggestion
msgstr "Déplacement vers le haut et la gauche sur la carte du monde"
```

File: files/lang/fr.po [hunk 12]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map up right"
-msgstr ""
+msgstr "Déplacement vers le haut et la droite dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4604]:
```suggestion
msgstr "Déplacement vers le haut et la droite sur la carte du monde"
```

File: files/lang/fr.po [hunk 13]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map down left"
-msgstr ""
+msgstr "Déplacement vers le bas et la gauche dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4607]:
```suggestion
msgstr "Déplacement vers le bas et la gauche sur la carte du monde"
```

File: files/lang/fr.po [hunk 14]
```diff
@@ -4593,28 +4586,28 @@ msgid "hotkey|restart campaign scenario"
...
 
 msgid "hotkey|world map down right"
-msgstr ""
+msgstr "Déplacement vers le bas et la droite dans la mappe du monde"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4610]:
```suggestion
msgstr "Déplacement vers le bas et la droite sur la carte du monde"
```

File: files/lang/fr.po [hunk 15]
```diff
@@ -4629,7 +4622,7 @@ msgid "hotkey|cast adventure spell"
 msgstr "Lancement de sorts d'aventure."
 
 msgid "hotkey|put hero to sleep"
-msgstr ""
+msgstr "Mettre le héros en mode \"dormir\""
```
Comment (FlorianBaba) [inline files/lang/fr.po:4625]: Not sur about the contexte on this one
```suggestion
msgstr "Mettre le héros en mode \"endormi\""
```

File: files/lang/fr.po [hunk 16]
```diff
@@ -4644,7 +4637,7 @@ msgid "hotkey|adventure options"
 msgstr "Options d'aventure"
 
 msgid "hotkey|puzzle map"
-msgstr ""
+msgstr "Ouvrir la mappe du puzzle"
```
Comment (FlorianBaba) [inline files/lang/fr.po:4640]:
```suggestion
msgstr "Visualiser le puzzle"
```

File: files/lang/fr.po [hunk 17]
```diff
@@ -9411,11 +9387,10 @@ msgstr ""
 msgid "Ballista of Quickness"
 msgstr "La baliste de rapidité"
 
-#, fuzzy
 msgid "The %{name} gives the hero's catapult one extra shot per combat round."
 msgstr ""
-"%{name} permet à votre catapulte de tirer un coup supplémentaire par round "
-"de combat."
+"%{name} permet à la catapulte du héros de tirer un coup supplémentaire par "
+"round de combat."
```
[the remark this answers is not in the document]
Comment (FlorianBaba) [reply files/lang/fr.po:9393]: I agree with @1GodRage , this would be my proposal:
```suggestion
"tour en combat."
```

File: files/lang/fr.po [hunk 18]
```diff
@@ -9957,9 +9920,8 @@ msgstr ""
 msgid "Pendant of Life"
 msgstr "Le pendentif de la vie"
 
-#, fuzzy
 msgid "The %{name} makes all of the hero's troops immune to death spells."
-msgstr "%{name} immunise vos troupes contre les sorts de la mort."
+msgstr "%{name} immunise les troupes du héros contre les sorts de la mort."
```
Comment (FlorianBaba) [inline files/lang/fr.po:9924]:
```suggestion
msgstr "%{name} immunise les troupes du héros contre les sorts de mort."
```

File: files/lang/fr.po [hunk 19]
```diff
@@ -10079,13 +10036,12 @@ msgstr ""
...
 
-#, fuzzy
 msgid ""
 "The %{name} increases the amount of terrain the hero reveals when "
 "adventuring by %{count} extra square."
 msgstr ""
-"%{name} augmente la quantité de terrain que votre héros révèle lorsqu'il "
-"part à l'aventure de %{count} carré supplémentaire."
+"%{name} augmente la quantité de terrain que le héros révèle lorsqu'il part à "
+"l'aventure de %{count} carré supplémentaire."
```
Comment (FlorianBaba) [inline files/lang/fr.po:10044]:
```suggestion
"l'aventure de %{count} cases supplémentaires."
```

File: files/lang/fr.po [hunk 20]
```diff
@@ -10478,15 +10425,14 @@ msgstr ""
...
 msgid ""
 "The %{name} combines the powers of the three Anduran artifacts. It provides "
 "maximum luck and morale for the hero's troops and gives the hero the Town "
 "Portal spell."
 msgstr ""
 "%{name} combine les pouvoirs des trois artéfacts d'Anduran. Il offre un "
-"maximum de chance et de moral à vos troupes et vous donne le sort portes du "
-"village."
+"maximum de chance et de moral aux troupes du héros et lui donne le sortilège "
+"portes du village."
```
Comment (FlorianBaba) [inline files/lang/fr.po:10435]:
```suggestion
"Portes du village."
```

File: files/lang/fr.po [hunk 21]
```diff
@@ -10631,13 +10575,12 @@ msgstr ""
...
 
-#, fuzzy
 msgid ""
 "The %{name} boosts the hero's troops' luck and morale by %{count} each in "
 "sea combat."
 msgstr ""
-"%{name} augmente votre chance et votre moral de %{count} points chacun dans "
-"les combats en mer."
+"%{name} augmente la chance et le moral de l'armée du héros de %{count} "
+"points chacun dans les batailles en mer."
```
Comment (FlorianBaba) [inline files/lang/fr.po:10583]:
```suggestion
"points chacun lors des batailles en mer."
```

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@zenseii , thanks a lot for this update!

**Pull request merged**